src/DataCollection.UWP/Views/IdentifiedFeaturePopup.xaml.cs

```diff
@@ -74,5 +77,21 @@ private async void OnPropertyChanged([CallerMemberName] string propertyName = nu
         }
 
         public event PropertyChangedEventHandler PropertyChanged;
+
+        /// <summary>
+        /// Event handler for user selecting to add a new attachment by capturing new media
+        /// </summary>
+        private async void CaptureMediaButton_Click(object sender, Windows.UI.Xaml.RoutedEventArgs e)
+        {
+            _identifiedFeatureViewModel.AttachmentsViewModel.NewAttachmentFile = await MediaHelper.RecordMediaAsync();
```

Comment:
I would check for exceptions here. If the user uses 'Esc' to close the file name dialog, the file gets saved as `.jpg`; doing that more than once throws an exception due to the duplicate name.

Reply:
Or just doing multiple attachments without changing the name will also throw an exception